Resources/views/Collector/elastica.html.twig

```diff
@@ -1,48 +1,60 @@
+            <span class="sf-toolbar-status">{{ collector.querycount }}</span>
+            {% if collector.querycount > 0 %}
+                <span class="sf-toolbar-info-piece-additional-detail">in {{ '%0.2f'|format(collector.time * 1000) }} ms</span>
+            {% endif %}
+        {% else %}
+            {{ include('@FOSElastica/Collector/icon.svg') }}
+            <span class="sf-toolbar-value">{{ collector.querycount }}</span>
+            {% if collector.querycount > 0 %}
```

Comment:
In the new toolbar, when a panel does not provide information, we don't display it. In this case, when there are 0 queries, this panel contains no information. If you want to follow this practice (which is not mandatory) you can change this:

```twig
{% include 'WebProfilerBundle:Profiler:toolbar_item.html.twig' with { 'link': profiler_url } %}
```

by this:

```twig
{% if collector.querycount > 0 %}
    {% include 'WebProfilerBundle:Profiler:toolbar_item.html.twig' with { 'link': profiler_url } %}
{% endif %}
```